### DISCUSSION TITLE
太记里面Wiki侧边栏的功能性优化

### DISCUSSION THREAD

**Discussion by Zacharia2** [tiddly-gittly/TidGi-Desktop#246] [Ideas]
太记里面的侧边栏对小白来说看着眼花，主要是在布局的排列上和不知名的功能上（自己感觉有些影响使用，会过度关注它是干什么用的，但是自己还不知道它们是做什么用的），或许像TiddlyWiki XP一样的排列会好一些吧。（太记的侧边栏有时候会自己滚动到中间去，比如在创建一个条目的时候，当我编辑好。在创建一个就需要在滚动一下侧边栏，找到按钮新建。）。

**Comment by linonetwo** [comment]
下次我把它去掉，那个插件还比较实验性…你可以通过删掉 flowtiwi-sidebar 插件来关闭这个效果。

**Reply by Zacharia2** [reply]
嗯嗯，它的排序也值得考虑一下，我感觉这样排会好一些，开启，文件目录，收藏，最近。或者收藏，文件目录，最近，开启
感觉上第二种体验或许会好一些。

**Reply by linonetwo** [reply]
现在这个 flowtiwi-sidebar 是可以拖动排序的，你可以试试呀

**Reply by linonetwo** [reply]
拖标题就可以排序，然后右下角可以改变瓷砖大小

**Reply by Zacharia2** [reply]
是在没有flowtiwi-sidebar的情况下啦(*^▽^*)

**Reply by Zacharia2** [reply]
谢谢Thanks♪(･ω･)ﾉO(∩_∩)O哈哈~
